proto/proto.go

```diff
@@ -0,0 +1,12 @@
+package proto
+
+type NodeId string
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type NodeID should have comment or be unexported